components/workflow-mgt/org.wso2.carbon.identity.workflow.mgt/src/main/java/org/wso2/carbon/identity/workflow/mgt/util/SQLConstants.java

```diff
@@ -57,13 +57,18 @@ public class SQLConstants {
     public static final String REQUEST_ID_COLUMN = "REQUEST_ID";
     public static final String CREATED_BY_COLUMN = "CREATED_BY";
 
+    public static  final String WORKFLOWID_COLUMN = "PARAM_VALUE";
+    public static final String TEMPLATEID_COLUMN = "TEMPLATE_ID";
 
     public static final String ADD_WORKFLOW_REQUEST_QUERY = "INSERT INTO WF_REQUEST(UUID, CREATED_BY, OPERATION_TYPE," +
             " CREATED_AT, UPDATED_AT, REQUEST, STATUS, TENANT_ID) VALUES (?, ?, ?, ?, ?, ?, ?, ?)";
 
     public static final String GET_WORKFLOW_REQUEST_QUERY = "SELECT UUID, REQUEST, STATUS, CREATED_BY FROM WF_REQUEST" +
             " WHERE UUID = ?";
-
+    public static final String GET_WORKFLOW_ID_QUERY = "SELECT PARAM_VALUE FROM WF_WORKFLOW_CONFIG_PARAM WHERE " +
+            "WORKFLOW_ID  = ?" +
+            " AND PARAM_NAME ='ExternalWorkflowIdentifier'";
```

Comment:
combine into single line

Reply:
PARAM_NAME = 'ExternalWorkflowIdentifier'.  also let's keep a space character after `=` operator

Reply:
addressed the comment ref:https://github.com/wso2/carbon-identity-framework/pull/4519/commits/ce9bfa08202eb691d2a4f87aebd30fe3445f0fd5